docs/FeatureManagement/Samples/TargetingFilter.modified.sample.json

```diff
@@ -0,0 +1,60 @@
+{
+    "feature_management": {
+        "feature_flags": [
+            {
+                "id": "ComplexTargeting",
+                "description": "A feature flag using a targeting filter, that will return true for Alice, Stage1, and 50% of Stage2, and false for Dave and Stage3. The default rollout percentage is 25%.",
+                "enabled": true,
+                "conditions": {
+                    "client_filters": [
+                        {
+                            "name": "Microsoft.Targeting",
+                            "parameters": {
+                                "Audience": {
+                                    "Users": [
+                                        "Alice"
+                                    ],
+                                    "Groups": [
+                                        {
+                                            "Name": "Stage1",
+                                            "RolloutPercentage": 100
+                                        },
+                                        {
+                                            "Name": "Stage2",
+                                            "RolloutPercentage": 50
+                                        }
+                                    ],
+                                    "DefaultRolloutPercentage": 25,
+                                    "Exclusion": {
+                                        "Users": ["Dave"],
+                                        "Groups": ["Stage3"]
+                                    }
+                                }
+                            }
+                        }
+                    ]
+                }
+            },
+            {
+                "id": "RolloutPercentageUpdate",
+                "description": "A feature flag using a targeting filter, that will return true 61% of the time. Changing to 62% makes the user Brittney true.",
```

Comment:
I think there's some mismatches with description/test function for this flag, specifically with the Brittney scenario